packages/components/src/tooltip/index.tsx

```diff
@@ -64,17 +87,30 @@ function Tooltip( props: TooltipProps ) {
 	}
 	computedPlacement = computedPlacement || 'bottom';
 
+	// Disable reason: the hook can not run conditionally.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
 	const tooltipStore = Ariakit.useTooltipStore( {
```

Comment:
This ESLint error can be circumvented by making it clearer that we're using a hook.

Something like this will do the job:

```suggestion
	const useAriakitTooltipStore = Ariakit.useTooltipStore;
	const tooltipStore = useAriakitTooltipStore( {
```

Feel free to also add a comment that explains why we're doing this.

Reply:
[`6e852d5` (#57202)](https://github.com/WordPress/gutenberg/pull/57202/commits/6e852d53c3e29de89422c072df2beab27017b925)

Reply:
We're reverting this now that it's no longer conditionally called: https://github.com/WordPress/gutenberg/pull/61858